src/reactionsystem.jl

```diff
@@ -1548,17 +1605,59 @@ function Base.convert(::Type{<:NonlinearSystem}, rs::ReactionSystem; name = name
     ists, ispcs = get_indep_sts(fullrs, remove_conserved)
```

Comment:
Here `ists` are no longer the independent unknowns now with your change, won't they be identical to the `ispcs`?

Reply:
Nevermind, I misunderstood your change.